src/plinder/data/utils/annotations/aggregate_annotations.py

```diff
@@ -623,13 +623,38 @@ def get_pocket_domains(self, chains_dict: dict[str, Chain]) -> dict[str, str]:
 
 
 class Entry(BaseModel):
-    pdb_id: str
-    release_date: str
-    oligomeric_state: str | None
-    determination_method: str | None
-    keywords: str | None
-    pH: str | None
-    resolution: float | None
+    pdb_id: str = Field(
+        default_factory=str,
+        help="RCSB PDB ID. See https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_entry.id.html",
+        serialization_alias="entry_pdb_id")
+    release_date: str = Field(
+        default_factory=str,
+        help="RCSB structure release date. See https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_database_PDB_rev.date_original.html",
+        serialization_alias="entry_release_date")
+    oligomeric_state: str | None = Field(
+        default_factory=str,
+        help="Author's provided description of quaternary structure in RCSB. See https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_pdbx_struct_assembly.oligomeric_details.html",
+        serialization_alias="entry_oligomeric_state")
+    determination_method: str | None = Field(
+        default_factory=str,
+        help="RCSB method of structure determination. See https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_exptl.method.html",
+        serialization_alias="entry_determination_method")
+    keywords: str | None = Field(
+        default_factory=str,
+        help="RCSB keywords describing the structure. See https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_struct_keywords.pdbx_keywords.html",
+        serialization_alias="entry_determination_method")
```

Comment:
Here and below are a couple erroneous copy+paste serialization aliases.

This pattern additionally reduces the need for customization in `format_entry|to_df` 👍